.changeset/quick-maps-check.md

```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/shopify-app-remix': minor
+'@shopify/shopify-api': minor
+---
+
+Test helpers and changes to enable automated unit and e2e testing for @shopify/shopify-app-remix
```

Comment:
These show up in the [release notes.](https://github.com/Shopify/shopify-app-js/releases/tag/%40shopify%2Fshopify-app-remix%403.1.0)

I like to have a short example to help folks understand who are skimming the notes to understand the new feature. And this is a valuable one, so I want to make sure folks understand the value! ⭐  This can just be copied from the docs you already wrote. And then a markdown link to the guide to learn more!

Reply:
That clarification helps! Thank you! I didn't know where the change notes flow through to.

One thing I'm still confused about though is how to link to the documentation correctly. Is there a relative path that I can use to point to the document from the change notes? Could you give me an example so that I can do it correctly?